themes/bootstrap3/templates/myresearch/fines.phtml

```diff
@@ -67,22 +67,28 @@
     ?>
     <table class="table table-striped">
       <caption class="sr-only"><?=$this->transEsc('Your Fines')?></caption>
+      <thead>
       <tr>
         <?php foreach ($columns as $header): ?>
-          <th><?=$this->transEsc($header)?></th>
+          <th scope="col"><?=$this->transEsc($header)?></th>
         <?php endforeach; ?>
       </tr>
+      </thead>
+      <tbody>
       <?php for ($row = 0; $row < $rowCount; $row++): ?>
         <tr>
           <?php foreach ($columns as $column): ?>
             <td><?=$tableData[$column][$row]?></td>
           <?php endforeach; ?>
         </tr>
       <?php endfor; ?>
-      <tr style="font-weight:bold">
-        <td colspan="<?=count($columns) - 1?>"><?=$this->transEsc('Total Balance Due')?></td>
-        <td><?=$this->safeMoneyFormat($totalDue / 100.00) ?></td>
+      </tbody>
+      <tfoot>
+      <tr>
+        <th colspan="<?=count($columns) - 1?>"><?=$this->transEsc('Total Balance Due')?></th>
+        <th><?=$this->safeMoneyFormat($totalDue / 100.00) ?></th>
```

Comment:
I think this might make more semantic sense as a td with bold styling, rather than a th, since this is a value rather than a heading.

Reply:
Since it summarizes the contents of all the cells above it, I think it should be `<th>`. (This is almost exactly the demo on the [MDN `<tfoot>` reference](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/tfoot).)

Reply:
@stephanieleary, sorry if I'm missing something, but in the MDN example you link to, the table cell in the tfoot containing the actual total value is a td, not a th, which seems to align with what I was originally suggesting. Is it possible we're just misunderstanding each other and not talking about the same cell? :-)

Reply:
(I definitely agree that the other footer cell, containing the label "Total Balance Due," unquestionably should be a th).

Reply:
No worries, glad we were able to get on the same page. :-) I'm going to run the test suite now just to be sure none of these changes require adjustments to integration tests, and if that goes well, I'll merge in a few minutes. Thanks again for these improvements!